### ISSUE TITLE
[homer] Issue on the Schema Json of the Helm Chart Homer on image.tag set with type null instead of string

### ISSUE THREAD

**Issue by cstisa** [djjudas21/charts#96]
### Describe the bug a clear and concise description of what the bug is.

Dear @djjudas21,

I think there is a typo on the line : https://github.com/djjudas21/charts/blob/main/charts/homer/values.schema.json#L81 the type should be set to string otherwise we can not override the version of the image tag if we don't want to use the specified version in AppVersion.

Here it says : https://github.com/djjudas21/charts/blob/main/charts/homer/README.md 

Thanks a lot 

Best Regards, Edouard Fazenda  for CSTI SA.

### What's your helm version?

8.1.12

### What's your kubectl version?

Client Version: v1.30.2 Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3 Server Version: v1.25.16

### Which chart?

homer

### What's the chart version?

latest

### What happened?

I have the following error : 

```
Error: values don't meet the specifications of the schema(s) in the following chart(s):
homer:
- image.tag: Invalid type. Expected: null, given: string
```

### What you expected to happen?

To ovveride the image.tag version in the values.

### How to reproduce it?

Specifiy in image.tag a version to override the AppVersion.

### Enter the command that you execute and failing/misfunctioning.

None

**Comment by djjudas21** [comment]
Thanks for reporting. I think this is a regression caused when I enabled [automatic schema generation](https://github.com/dadav/helm-schema) in 7a5ec6c32cfc3d6bbd6d8a32c9c6392e65f2c206 which caused the auto-generation in 479b31ba3532c5fdca777c3367c82f8c1ecd1660.

I'll have a look at this today if I get time... it probably just needs the default changing from null to an empty string, or maybe a metadata override.

**Comment by cstisa** [comment]
Hi @djjudas21 ,

Yes thank you, no worries not urgent, but thanks for having a look :) 

Best Regards, Edouard Fazenda.

**Comment by djjudas21** [comment]
This should be fixed in Homer chart version v8.1.15. Please let me know if any trouble. Thanks!
